keda/templates/deployment.yaml

```diff
@@ -41,30 +49,48 @@ spec:
           - name: grpc-certs
             mountPath: /grpccerts
           {{- end }}
+          resources:
+            {{- toYaml .Values.resources | nindent 12 }}
         - name: {{ .Values.operatorName }}-metrics-apiserver
-          image: {{ .Values.image.metricsAdapter }}
+          image: "{{ .Values.image.metricsAdapter }}:{{ .Chart.AppVersion }}"
```

Comment:
Ok for you if we add this @ahmelsayed? It's fine for me.

Reply:
Thanks Tom, I just realized that I added the incorrect ports in the deployment.yaml file, if you like I can add this change in a new commit and pull request or if you can help me to replace it also is good for me:

This is the correct piece that should be in deployment.yaml line 65-71:
```
          ports:
            - containerPort: {{ .Values.service.portHttpsTarget }}
              name: https
              protocol: TCP
            - containerPort: {{ .Values.service.portHttpTarget }}
              name: http
              protocol: TCP
```
Thanks,
RC

Reply:
Thanks Tom, I just realized that I added the incorrect ports in the deployment.yaml file, if you like I can add this change in a new commit and pull request or if you can help me to replace it also is good for me.

This is the correct piece that should be in deployment.yaml line 65-71:
```
          ports:
            - containerPort: {{ .Values.service.portHttpsTarget }}
              name: https
              protocol: TCP
            - containerPort: {{ .Values.service.portHttpTarget }}
              name: http
              protocol: TCP
```
Thanks,
RC

Reply:
Thanks Tom, I just realized that I added the incorrect ports in the deployment.yaml file, I going to close this PR and send a new one with the changes updated.